index.html

```diff
@@ -0,0 +1,515 @@
+          </div>
+          <div class="category-latest"><span class="latest">Latest</span>
+          <h4 class="cat-latest-heading">An introduction to Bitcoin and cryptocurrency mining pools</h4><img alt="" src="#"></div>
+          <ul class="category-items">
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+          </ul>
+        </div>
+        <div class="single-category">
+          <h2 class="category-heading">Placeholder</h2>
+          <div class="category-featured">
+            <span class="most-popular">Most popular</span>
+            <h3 class="featured-title">Anonymous Bitcoin miners are taking over the network</h3><span>David Canellis &#8226; 18 hours ago</span>
+          </div>
+          <div class="category-latest"><span class="latest">Latest</span>
+          <h4 class="cat-latest-heading">An introduction to Bitcoin and cryptocurrency mining pools</h4><img alt="" src="#"></div>
+          <ul class="category-items">
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+          </ul>
+        </div>
+        <div class="single-category">
+          <h2 class="category-heading">Placeholder</h2>
+          <div class="category-featured">
+            <span class="most-popular">Most popular</span>
+            <h3 class="featured-title">Anonymous Bitcoin miners are taking over the network</h3><span>David Canellis &#8226; 18 hours ago</span>
+          </div>
+          <div class="category-latest"><span class="latest">Latest</span>
+          <h4 class="cat-latest-heading">An introduction to Bitcoin and cryptocurrency mining pools</h4><img alt="" src="#"></div>
+          <ul class="category-items">
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+            <li class="category-item">BBC News website spoofed by Bitcoin scammers</li>
+          </ul>
+        </div>
+
+      </section>
+
+    <section class="latest-deals">
```

Comment:
Again, match the appearance of the original. Pictures should be squares. Use the same colors. Check the font size and style. Also, the spaces between elements.